README.md

```diff
@@ -102,11 +118,14 @@ annotation to your .proto file
 
 3. Generate gRPC stub
 
+   Suppose `ROOT_TO_PROTO` is where your proto files store and `DEST` is where you want put generated files in, do:  
+
    ```sh
-   protoc -I/usr/local/include -I. \
+   protoc -I/usr/local/include \
+     -I${ROOT_TO_PROTO} \
      -I$GOPATH/src \
      -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
-     --go_out=plugins=grpc:. \
+     --go_out=plugins=grpc:${DEST} \
```

Comment:
While we're doing this, we should add `paths=source_relative`. It simplifies generated file output greatly.

Reply:
I think it is better to leave users to decide where to install. B.W.T, for newbee of golang like me, I didn't even realize immediately that I can change output directory. For me, I would like to put all generated files in build (I use cmake to build cpp/python/golang app) so that they are all covered by dependencies resolution system.

Reply:
My comment doesn't contradict that. We should still add `paths=source_relative` as an option to the go_out command like so: 
```
--go_out=plugins=grpc,paths=source_relative:${DEST}
```

Reply:
copy it.